### PR TITLE
Map some sundry names

### PR DESCRIPTION
**Pull request by bluebear94**

### CODE REVIEW

**Review by liach — APPROVED** [review]

**Review by apple502j — APPROVED** [review]

File: mappings/net/minecraft/world/gen/feature/TreePlacedFeatures.mapping [hunk 1]
```diff
@@ -26,5 +26,7 @@ CLASS net/minecraft/class_6818 net/minecraft/world/gen/feature/TreePlacedFeature
 	FIELD field_36113 FANCY_OAK_BEES_002 Lnet/minecraft/class_5321;
 	FIELD field_38814 MANGROVE_CHECKED Lnet/minecraft/class_5321;
 	FIELD field_38815 TALL_MANGROVE_CHECKED Lnet/minecraft/class_5321;
+	FIELD field_42962 CHERY_BEES_005 Lnet/minecraft/class_5321;
```
Comment (haykam821) [inline mappings/net/minecraft/world/gen/feature/TreePlacedFeatures.mapping]:
```suggestion
	FIELD field_42962 CHERRY_BEES_005 Lnet/minecraft/class_5321;
```

File: mappings/net/minecraft/world/gen/trunk/GiantTrunkPlacer.mapping [hunk 2]
```diff
@@ -1,5 +1,14 @@
 CLASS net/minecraft/class_5214 net/minecraft/world/gen/trunk/GiantTrunkPlacer
 	FIELD field_24969 CODEC Lcom/mojang/serialization/Codec;
 	METHOD method_27399 setLog (Lnet/minecraft/class_3746;Ljava/util/function/BiConsumer;Lnet/minecraft/class_5819;Lnet/minecraft/class_2338$class_2339;Lnet/minecraft/class_4643;Lnet/minecraft/class_2338;III)V
+		ARG 1 world
+		ARG 2 replacer
+		ARG 3 random
+		ARG 4 tmp
+		ARG 5 config
+		ARG 6 basePos
```
Comment (haykam821) [inline mappings/net/minecraft/world/gen/trunk/GiantTrunkPlacer.mapping]:
```suggestion
		ARG 6 startPos
```

File: mappings/net/minecraft/world/gen/trunk/GiantTrunkPlacer.mapping [hunk 3]
```diff
@@ -1,5 +1,14 @@
 CLASS net/minecraft/class_5214 net/minecraft/world/gen/trunk/GiantTrunkPlacer
 	FIELD field_24969 CODEC Lcom/mojang/serialization/Codec;
 	METHOD method_27399 setLog (Lnet/minecraft/class_3746;Ljava/util/function/BiConsumer;Lnet/minecraft/class_5819;Lnet/minecraft/class_2338$class_2339;Lnet/minecraft/class_4643;Lnet/minecraft/class_2338;III)V
+		ARG 1 world
+		ARG 2 replacer
+		ARG 3 random
+		ARG 4 tmp
```
Comment (haykam821) [inline mappings/net/minecraft/world/gen/trunk/GiantTrunkPlacer.mapping]: Perhaps simply `pos`? I'm not sure we have a standard for this.

```suggestion
		ARG 4 pos
```

File: mappings/net/minecraft/util/math/GivensPair.mapping [hunk 4]
```diff
@@ -6,5 +6,11 @@ CLASS net/minecraft/class_8218 net/minecraft/util/math/GivensPair
 	METHOD method_49727 normalize (FF)Lnet/minecraft/class_8218;
 		ARG 0 a
 		ARG 1 b
+	METHOD method_49728 setXRotation (Lorg/joml/Matrix3f;)Lorg/joml/Matrix3f;
+	METHOD method_49729 setXRotation (Lorg/joml/Quaternionf;)Lorg/joml/Quaternionf;
```
Comment (haykam821) [inline mappings/net/minecraft/util/math/GivensPair.mapping]:
```suggestion
	METHOD method_49728 setRotationX (Lorg/joml/Matrix3f;)Lorg/joml/Matrix3f;
	METHOD method_49729 setRotationX (Lorg/joml/Quaternionf;)Lorg/joml/Quaternionf;
```

File: mappings/net/minecraft/util/math/GivensPair.mapping [hunk 5]
```diff
@@ -6,5 +6,11 @@ CLASS net/minecraft/class_8218 net/minecraft/util/math/GivensPair
...
 	METHOD method_49730 cosDouble ()F
+	METHOD method_49731 setYRotation (Lorg/joml/Matrix3f;)Lorg/joml/Matrix3f;
+	METHOD method_49732 setYRotation (Lorg/joml/Quaternionf;)Lorg/joml/Quaternionf;
```
Comment (haykam821) [inline mappings/net/minecraft/util/math/GivensPair.mapping]:
```suggestion
	METHOD method_49731 setRotationY (Lorg/joml/Matrix3f;)Lorg/joml/Matrix3f;
	METHOD method_49732 setRotationY (Lorg/joml/Quaternionf;)Lorg/joml/Quaternionf;
```

File: mappings/net/minecraft/util/math/GivensPair.mapping [hunk 6]
```diff
@@ -6,5 +6,11 @@ CLASS net/minecraft/class_8218 net/minecraft/util/math/GivensPair
...
 	METHOD method_49733 sinDouble ()F
+	METHOD method_49734 setZRotation (Lorg/joml/Matrix3f;)Lorg/joml/Matrix3f;
+	METHOD method_49735 setZRotation (Lorg/joml/Quaternionf;)Lorg/joml/Quaternionf;
```
Comment (haykam821) [inline mappings/net/minecraft/util/math/GivensPair.mapping]:
```suggestion
	METHOD method_49734 setRotationZ (Lorg/joml/Matrix3f;)Lorg/joml/Matrix3f;
	METHOD method_49735 setRotationZ (Lorg/joml/Quaternionf;)Lorg/joml/Quaternionf;
```

File: mappings/net/minecraft/util/math/MathHelper.mapping [hunk 7]
```diff
@@ -340,6 +340,10 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 		ARG 0 a
 		ARG 2 b
 		ARG 4 c
+	METHOD method_34945 (Lnet/minecraft/class_243;Lnet/minecraft/class_243;Lnet/minecraft/class_238;)Z
```
Comment (haykam821) [inline mappings/net/minecraft/util/math/MathHelper.mapping:343]: Could this method be named?

File: mappings/net/minecraft/util/math/MathHelper.mapping [hunk 8]
```diff
@@ -340,6 +340,10 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
...
+		ARG 0 origin
+		ARG 1 direction
+		ARG 2 aabb
```
Comment (haykam821) [inline mappings/net/minecraft/util/math/MathHelper.mapping]:
```suggestion
		ARG 2 box
```

File: mappings/net/minecraft/fluid/FlowableFluid.mapping [hunk 9]
```diff
@@ -80,6 +86,9 @@ CLASS net/minecraft/class_3609 net/minecraft/fluid/FlowableFluid
 		ARG 6 fromPos
 		ARG 7 fromState
 		ARG 8 fluidState
+	METHOD method_15747 packXZOffset (Lnet/minecraft/class_2338;Lnet/minecraft/class_2338;)S
```
Comment (mschae23) [inline mappings/net/minecraft/fluid/FlowableFluid.mapping:89]: `packXzOffset` or `packOffsetXz`

Comment (Shnupbups) [reply mappings/net/minecraft/fluid/FlowableFluid.mapping:89]: No, how it is is right. If XZ was an initialism or a single word, it'd be `Xz`, but as they are separate things, `XZ` is correct.

File: mappings/net/minecraft/util/math/MathHelper.mapping (hunk 7 above)
Comment (bluebear94) [reply mappings/net/minecraft/util/math/MathHelper.mapping:343]: I don’t know – it’s not used anywhere in the code. It seems to be some kind of line–box collision detection algorithm, but I can’t find any code in the wild that looks like it.

File: mappings/net/minecraft/util/math/Box.mapping [hunk 10]
```diff
@@ -84,7 +84,7 @@ CLASS net/minecraft/class_238 net/minecraft/util/math/Box
 		ARG 1 x
 		ARG 3 y
 		ARG 5 z
-	METHOD method_1013 isValid ()Z
+	METHOD method_1013 isNaN ()Z
 		COMMENT Checks if any of the coordinates of this box is {@linkplain
```
Comment (haykam821) [inline mappings/net/minecraft/util/math/Box.mapping]:
```suggestion
		COMMENT Checks if any of the coordinates of this box are {@linkplain
```

File: mappings/net/minecraft/fluid/FlowableFluid.mapping (hunk 9 above)
Comment (haykam821) [reply mappings/net/minecraft/fluid/FlowableFluid.mapping:89]: Shouldn't it be `packOffsetXZ` then?

```suggestion
	METHOD method_15747 packOffsetXZ (Lnet/minecraft/class_2338;Lnet/minecraft/class_2338;)S
```

**Review by apple502j — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**